ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -196,10 +196,8 @@ set_local_repo() {
     local version
     version="$(lsb_release -sr)"
     version=${version%%.*}
-    if [ "$repo_server" = "artifactory" ] &&
-       [ -z "$(rpm_test_version)" ] &&
-       [[ ${CHANGE_TARGET:-$BRANCH_NAME} != weekly-testing* ]] &&
-       [[ ${CHANGE_TARGET:-$BRANCH_NAME} != provider-testing* ]]; then
+    if [ "$repo_server" = "artifactory" ] && [ -z "$(rpm_test_version)" ] &&
+       [[ ! ${CHANGE_TARGET:-$BRANCH_NAME} =~ ^[a-z]+-testing ]]; then
```

Comment:
I see a choice/decision has been made.  Given that, can you expand the RE to include a few more values:

```suggestion
       [[ ! ${CHANGE_TARGET:-$BRANCH_NAME} =~ ^[-0-9A-Za-z]+-testing ]]; then
```

And of course (and more importantly), this change needs to be made in the `ci/provisioning/` dirs in the actual target branches -- `master`, `release/*`, etc. since those are the branches that are actually checked out and the provisioning happens from for these _testing_ branches).  This file/dir is only used in the testing of pipeline-lib changes.  As you can see, this is why I want to make this change such that it scales out to any future branches of this sort without having to keep modifying this hunk of code in the growing number of branches it will apply to.

Reply:
Thanks Brian,  I'll update the pr

The ci/provisioning changes are in PR-9911